blog/electron-23-0.md

```diff
@@ -0,0 +1,91 @@
+    * [New in DevTools 109](https://developer.chrome.com/blog/new-in-devtools-109/)
+* Node.js `18.12.1`
+    * [Node 18.12.1 blog post](https://nodejs.org/en/blog/release/v18.12.1/)
+* V8 `11.0`
+
+### New Features
+
+* Added `label` property to `Display` objects. [#36933](https://github.com/electron/electron/pull/36933) 
+* Added an `app.getPreferredSystemLanguages()` API to return the user's system languages. [#36035](https://github.com/electron/electron/pull/36035) 
+* Added support for WebUSB. [#36289](https://github.com/electron/electron/pull/36289) 
+* Added support for `serialPort.forget()` as well as a new event `serial-port-revoked` emitted when a given origin is revoked. [#35310](https://github.com/electron/electron/pull/35310)
+* Adds new `setHiddenInMissionControl` API to allow developers to opt out of mission control on macOS. [#36092](https://github.com/electron/electron/pull/36092)
+
+## Windows 7/8/8.1 Support Update
+
+Electron 23 no longer supports Windows 7/8/8.1. Electron follows the planned Chromium deprecation policy, which will [deprecate Windows 7/8/8.1 , as well as Windows Server 2012 and 2012 R2 support in Chromium 109 (read more here)](https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en).
+
+## Breaking API Changes
+
+Below are breaking changes introduced in Electron 23. You can read more about these changes and future changes on the [Planned Breaking Changes](https://github.com/electron/electron/blob/main/docs/breaking-changes.md) page.
+
+#### Removed: `BrowserWindow scroll-touch-* events`
+
+The deprecated scroll-touch-begin, scroll-touch-end and scroll-touch-edge events on BrowserWindow have been removed. Instead, use the newly available input-event event on WebContents.
+
+```diff
+// Removed in Electron 23.0
+-win.on('scroll-touch-begin', scrollTouchBegin)
+-win.on('scroll-touch-edge', scrollTouchEdge)
+-win.on('scroll-touch-end', scrollTouchEnd)
+
+// Replace with
++win.webContents.on('input-event', (_, event) => {
++  if (event.type === 'gestureScrollBegin') {
++    scrollTouchBegin()
++  } else if (event.type === 'gestureScrollUpdate') +{
++    scrollTouchEdge()
++  } else if (event.type === 'gestureScrollEnd') {
++    scrollTouchEnd()
++  }
++})
+```
+
+## End of Support for 20.x.y
+
+Electron 20.x.y has reached end-of-support as per the project's [support policy](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#version-support-policy). Developers and applications are encouraged to upgrade to a newer version of Electron.
+
+| E20 (Aug'22) | E21 (Sep'22) | E22 (Nov'22) | E23 (Jan'23) | E24 (Apr'23) |
+| ------------ | ------------ | ------------ | ------------ | ------------ |
+| 20.x.y       | 21.x.y       | 22.x.y       | 23.x.y       | 24.x.y       |
+| 19.x.y       | 20.x.y       | 21.x.y       | 22.x.y       | 23.x.y       
+| 18.x.y       | 19.x.y       | 20.x.y       | 21.x.y       | 22.x.y       |
```

Comment:
Since we're deprecating E20, I think we can make the table more future focused:

```suggestion
| E22 (Nov'22) | E23 (Feb'23) | E24 (Apr'23) | E25 (May'23) | E26 (Aug'23) |
| ------------ | ------------ | ------------ | ------------ | ------------ |
| 22.x.y       | 23.x.y       | 24.x.y       | 25.x.y       | 26.x.y       |
| 21.x.y       | 22.x.y       | 23.x.y       | 24.x.y       | 25.x.y       
| 20.x.y       | 21.x.y       | 22.x.y       | 23.x.y       | 24.x.y       |
```